.github/workflows/packaging.yaml

```diff
@@ -78,7 +78,7 @@ jobs:
           key: test-${{ runner.os }}-${{ env.GO_VERSION }}-go-${{ hashFiles('go.sum', 'Makefile') }}
           restore-keys: test-${{ runner.os }}-${{ env.GO_VERSION }}-go-
 
-      - name: "Build Windows Installer (MSI)"
+      - name: "Build packages (.msi, .deb [amd64, arm64], .rpm [x86_64, aarch64])"
```

Comment:
If this is too much, please scream at me what the perfect description for this is. (needed to change since it is not all about "Windows Installer (MSI)" anymore).